source/PyMaterialX/PyDefinition.cpp

```diff
@@ -18,9 +18,14 @@ void bindPyDefinition(py::module& mod)
         .def("setNodeString", &mx::NodeDef::setNodeString)
         .def("hasNodeString", &mx::NodeDef::hasNodeString)
         .def("getNodeString", &mx::NodeDef::getNodeString)
-        .def("getImplementation", &mx::NodeDef::getImplementation)
+        .def("getImplementation", (mx::InterfaceElementPtr (mx::NodeDef::*)(const std::string &) const) &mx::NodeDef::getImplementation,
+            py::arg("target") = mx::EMPTY_STRING)
+        .def("getImplementation", (mx::InterfaceElementPtr(mx::NodeDef::*)(const std::string &, const std::string &) const) &mx::NodeDef::getImplementation,
+            py::arg("language"), py::arg("target"))
+        .def("requiresImplementation", &mx::NodeDef::requiresImplementation)
         .def("getInstantiatingShaderRefs", &mx::NodeDef::getInstantiatingShaderRefs)
-        .def_readonly_static("CATEGORY", &mx::NodeDef::CATEGORY);
+        .def_readonly_static("CATEGORY", &mx::NodeDef::CATEGORY)
```

Comment:
Noticed a few statics not put into the Python api so just adding them

Reply:
Good call, and thanks for adding these!